### PR TITLE
Cont. Account Critical Updates

### PR DESCRIPTION
**Pull request by itsalyse**
This PR updates the following articles in the Account category marked Critical: 

Managing Multiple Accounts Per User - https://docs.google.com/document/d/1dbBQLMTujHdirXVrGUPYlR8YYW9G0Iaz7XRJVZprH6Q/edit

Deleting Yourself As A User - https://docs.google.com/document/d/1dbBQLMTujHdirXVrGUPYlR8YYW9G0Iaz7XRJVZprH6Q/edit#heading=h.ymti5ckbczmt

Managing Seats - https://docs.google.com/document/d/1SdMt5fMs0Ze9ByMdeFVHTaB90llWigVC7LbO4rioE7E/edit

Securing Your DNSimple Account - https://docs.google.com/document/d/1Zw88oRueiQWuujD5-NfVg0V3pRo53dr-YZF_bnWMah8/edit#heading=h.qjk88tjgk00j

Related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/account-multi.md [hunk 1]
```diff
@@ -17,39 +17,43 @@ categories:
...
 
-If you want to add another account to your user profile:
+To create another account under your user profile:
 
 <div class="section-steps" markdown="1">
-1. Visit the settings for your user profile by clicking the <label>Settings</label> link in the menu.
+1. Click <label>User settings</label> at the top right of the screen.
 
-    ![Settings link](/files/access-user-settings.jpg)
+    **new screenshot**
```
Comment (itsalyse) [inline content/articles/account-multi.md:31]: https://drive.google.com/file/d/1Lxuvl7kHKDeaQtdbx7GTn6cam8TBq2pM/view?usp=sharing

File: content/articles/account-multi.md [hunk 2]
```diff
@@ -17,39 +17,43 @@ categories:
...
-1. Once there, follow the <label>Add new account</label> link.
+1. Scroll down to the **Accounts** card, and click <label>Add</label>.
 
     ![Add New Account link](/files/add-new-account-link.png)
 
-1. You'll see a form you should fill in with the details for your new account.
+1. Fill in the form with the details for your new account. Click <label>Create account</label>.
 
-    ![Add New Account form](/files/add-new-account-form.png)
+    **New screenshot**
```
Comment (itsalyse) [inline content/articles/account-multi.md:39]: https://drive.google.com/file/d/1ZpLfKqwVh24IvYQq1S5gY0J5Uv5nIU7f/view?usp=sharing

File: content/articles/account-multi.md [hunk 3]
```diff
@@ -17,39 +17,43 @@ categories:
...
 ## Switching active accounts
 
-Switching account context is easy. Select the account you'd like to use from the menu:
+Select the account you want to view from the top bar of your dashboard, and you'll be taken to the Domain List for that account. 
+
+**New screenshot** 
+
+From your Domain List, click **_Account_** in the top right to view details for the selected account.
+
+**new screenshot**
```
Comment (itsalyse) [inline content/articles/account-multi.md:55]: https://drive.google.com/file/d/1SZyep5kJKk_BZ4jET-HRrXlJztPRwxqk/view?usp=sharing

File: content/articles/close-account.md [hunk 4]
```diff
@@ -19,16 +19,25 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
 <div class="section-steps" markdown="1">
 ##### To permanently delete yourself as a user, along with all your domains and records:
 
+1. [Unsubscribe your accounts from your plan(s) at DNSimple](/articles/cancel-subscription).
+
 1. Go to [your User Settings page](https://dnsimple.com/user):
 
-    ![Navigate to user settings page](/files/access-user-settings.jpg)
+    **New screenshot**
```
Comment (itsalyse) [inline content/articles/close-account.md:26]: https://drive.google.com/file/d/1Lxuvl7kHKDeaQtdbx7GTn6cam8TBq2pM/view?usp=sharing

File: content/articles/close-account.md [hunk 5]
```diff
@@ -19,16 +19,25 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
...
+
 1. Go to [your User Settings page](https://dnsimple.com/user):
 
-    ![Navigate to user settings page](/files/access-user-settings.jpg)
+    **New screenshot** 
 
-1. Click on the <label>Delete user</label> button at the bottom of the page:
+1. Click <label>Delete user</label> at the bottom of the page:
 
      ![Delete user](/files/delete-user.png)
```
Comment (itsalyse) [inline content/articles/close-account.md:30]: https://drive.google.com/file/d/15SSTEd-8PUcVsdHMLPIB9gGg44ircuqH/view?usp=sharing

File: content/articles/close-account.md [hunk 6]
```diff
@@ -19,16 +19,27 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
...
+1. [Unsubscribe your accounts from your plan(s) at DNSimple](/articles/cancel-subscription).
+
 1. Go to [your User Settings page](https://dnsimple.com/user):
 
-    ![Navigate to user settings page](/files/access-user-settings.jpg)
+    **New screenshot** 
+
+1. Click <label>Delete user</label> at the bottom of the page:
+
+    **New screenshot**
```
Comment (itsalyse) [inline content/articles/close-account.md:30]: https://drive.google.com/file/d/15SSTEd-8PUcVsdHMLPIB9gGg44ircuqH/view?usp=sharing

File: content/articles/close-account.md [hunk 7]
```diff
@@ -19,16 +19,29 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
...
 
-     ![Delete user](/files/delete-user.png)
+   <info>
+   The <label>Delete user</label> button will only be available if you have no active DNSimple subscriptions.
+   </info>
+   
+1. Include your feedback, or write anything you like in the provided field, then click **I understand the consequences, close this user and account**. The field cannot be left blank.
 
-   The <label>Delete user</label> button will only show up if you have no active DNSimple subscriptions.
+   **New screenshot**
```
Comment (itsalyse) [inline content/articles/close-account.md:38]: https://drive.google.com/file/d/1OFQYmEOe8REyTN6yw4W-Y43BzQQopDNN/view?usp=sharing

File: content/articles/account-securing.md [hunk 8]
```diff
@@ -5,13 +5,40 @@ categories:
...
-- For an extra level of security, [enable multi-factor authentication](/articles/multi-factor-authentication).
+* TOC
+{:toc}
+
+---
+
+Domains are some of the most important assets for any organization, business, or personal project. We do everything we can to keep your domains and information secure, but there are steps you can take to further secure your DNSimple account.
+
+<alert> 
+If you've noticed fraudulent activity, or suspect someone has taken over your account, contact security@dnsimple.com.
```
Comment (itsalyse) [inline content/articles/account-securing.md:20]: This is from the security policy, but lmk if that's not right

File: content/articles/managing-seats.md [hunk 9]
```diff
@@ -45,14 +45,18 @@ Below is an example of decreasing seats when the number of account members excee
 To invite new members to your account, follow these steps:
 
 1. Access your account settings.
-1. Select the "Member and Seats" tab.
-1. Click on "Invite new member" in the "Members" card.
-1. If additional seats are required for new members, you'll see a "Add seats to invite new members" button. Click it to allocate more seats.
+1. Select the **Member and Seats** tab.
+1. Click **Invite new member** in the **Members** card.
+1. If additional seats are required for new members, you'll see an **Add seats to invite new members** button. Click it to allocate more seats.
```
Comment (itsalyse) [inline content/articles/managing-seats.md:50]: Screenshot between 49 & 50 

https://drive.google.com/file/d/1_ZZCIJlRpOt82UBCC0P-Oy1vjn8he2YO/view?usp=sharing

File: content/articles/managing-seats.md [hunk 10]
```diff
@@ -14,27 +14,27 @@ categories:
...
 
 ## Increasing the number of seats
 
 To increase the number of seats, follow these steps:
 
 1. Navigate to your account settings.
-1. Select the "Member and Seats" tab.
-1. Click "Manage" within the "Seats" card.
+1. Select the **Member and seats** tab.
+1. Click **Manage** within the **Seats** card.
```
Comment (itsalyse) [inline content/articles/managing-seats.md:25]: Screenshot between 24 & 25 

https://drive.google.com/file/d/12PhnVISraqB_1vAniQWApMwnqfVUFkdi/view?usp=sharing

File: content/articles/managing-seats.md [hunk 11]
```diff
@@ -14,27 +14,27 @@ categories:
...
 1. Choose the desired number of new seats.
-1. Click "Update seats" to save your changes.
+1. Click **Update seats** to save your changes.
```
Comment (itsalyse) [inline content/articles/managing-seats.md:28]: Screenshot https://drive.google.com/file/d/1uJo_hwJfixbh71saXc-TtpTs_wrw6-At/view?usp=sharing

File: content/articles/managing-seats.md [hunk 12]
```diff
@@ -14,27 +14,27 @@ categories:
```
Comment (itsalyse) [inline content/articles/managing-seats.md:14]: This has been one of the tricker ones to update, I'm trying to navigate it through the lens of seat = spot in the account, team member = person, but let me know if anything is still unclear here.

File: content/articles/account-securing.md (hunk 8 above)
Comment (stephanoodle) [reply content/articles/account-securing.md:20]: Customers will most likely have eyes on their account faster if they write to support@dnsimple.com. We investigate and then we internally flag for security/compliance review.

File: content/articles/managing-seats.md (hunk 12 above)
Comment (stephanoodle) [reply content/articles/managing-seats.md:14]: It's much clearer than before.

**Review by stephanoodle — APPROVED** [review]

File: content/articles/account-securing.md (hunk 8 above)
Comment (Pemacaa) [reply content/articles/account-securing.md:20]: Agree with Stephanie.

**Review by Pemacaa — APPROVED** [review]

File: content/articles/account-securing.md (hunk 8 above)
Comment (dperrymorrow) [reply content/articles/account-securing.md]: i have updated this ^^

**Review by dperrymorrow — APPROVED** [review]
I have added the images, compressed them, and updated the email link to point to support rather than security as suggested

File: content/articles/close-account.md (hunk 4 above)
Comment (itsalyse) [reply content/articles/close-account.md:26]: @dperrymorrow These screenshots were not added. Please reopen and add the screenshots.

File: content/articles/close-account.md (hunk 6 above)
Comment (itsalyse) [reply content/articles/close-account.md:30]: @dperrymorrow These screenshots were not added. Please reopen and add the screenshots.

File: content/articles/close-account.md (hunk 7 above)
Comment (itsalyse) [reply content/articles/close-account.md:38]: @dperrymorrow These screenshots were not added. Please reopen and add the screenshots.

**Pull request merged**